source/Application/Model/Category.php

```diff
@@ -985,8 +985,15 @@ protected function _update()
     protected function _setFieldData($sFieldName, $sValue, $iDataType = \OxidEsales\Eshop\Core\Field::T_TEXT)
     {
         //preliminary quick check saves 3% of execution time in category lists by avoiding redundant strtolower() call
-        if ($sFieldName[2] == 'l' || $sFieldName[2] == 'L' || (isset($sFieldName[16]) && ($sFieldName[16] == 'l' || $sFieldName[16] == 'L'))) {
-            if ('oxlongdesc' === strtolower($sFieldName) || 'oxcategories__oxlongdesc' === strtolower($sFieldName)) {
+        if (($sFieldNameIndex2 = $sFieldName[2]) === 'l'
```

Comment:
Please don't use hungarian notations anymore. Check our coding styles agreements.

Reply:
Please don't make complex conditions even more complex :) Try to improve the readability instead.